### ISSUE TITLE
Accessibility audit: (JAWS) Slack channel link not public

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#313]
## Describe the issue
The slack channel link is only available to DWP slack members, so including the link on a public facing site just adds a barrier.

## Fix recommendations

- [x] Remove the link to the slack channel, but include a reference with some context
- [ ] Or, replace slack reference with a GitHub issue link

## Related stories

- [Accessibility audit: (JAWS) Get in touch channel # announcement](https://github.com/dwp/design-system/issues/312)

**Comment by martinwake** [comment]
Discussed with @simoneduca and I recommended removing the link to the Slack channel from the "Get in touch" panel. While we think it's still OK to link to private channels when needed it probably isn't the right channel for general public feedback and comments.
